TechSupport-Session-Manager/src/main/java/com/github/RevatureTechSupport/TechSupportSessionManager/controller/QueueController.java

```diff
@@ -0,0 +1,77 @@
+package com.github.RevatureTechSupport.TechSupportSessionManager.controller;
+
+import com.github.RevatureTechSupport.TechSupportSessionManager.Repository.QueueRepository;
+import com.github.RevatureTechSupport.TechSupportSessionManager.domain.Issue;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.server.ServerRequest;
+import org.springframework.web.reactive.function.server.ServerResponse;
+import reactor.core.publisher.Mono;
+
+import java.net.URI;
+import java.sql.Timestamp;
+import java.util.UUID;
+
+@Component
+public class QueueController {
+    static Logger logger = LoggerFactory.getLogger(QueueController.class);
+
+    @Autowired
+    private QueueRepository queueRepository;
+
+    // Get All issues available in the queue
+    public Mono<ServerResponse> getAllIssues(ServerRequest request) {
```

Comment:
all others are req so changing for consistency
```suggestion
    public Mono<ServerResponse> getAllIssues(ServerRequest req) {
```